datasets/narrativeqa_manual/narrativeqa_manual.py

```diff
@@ -0,0 +1,248 @@
+
+
+_HOMEPAGE = "https://deepmind.com/research/publications/narrativeqa-reading-comprehension-challenge"
+_LICENSE = "https://github.com/deepmind/narrativeqa/blob/master/LICENSE"
+
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://github.com/deepmind/narrativeqa"
+_URLS = {
+    "documents": "https://raw.githubusercontent.com/deepmind/narrativeqa/master/documents.csv",
+    "summaries": "https://raw.githubusercontent.com/deepmind/narrativeqa/master/third_party/wikipedia/summaries.csv",
+    "qaps": "https://raw.githubusercontent.com/deepmind/narrativeqa/master/qaps.csv",
+}
+
+
+class NarrativeqaManual(datasets.GeneratorBasedBuilder):
+    """The NarrativeQA Manual dataset"""
+
+    VERSION = datasets.Version("1.0.0")
+
+    @property
+    def manual_download_instructions(self):
+        return """ You need to manually download the stories for this dataset using the script provided by the authors \
+                (https://github.com/deepmind/narrativeqa/blob/master/download_stories.sh). Running the shell script creates a folder named "tmp"\
+                in the root directory and downloads the stories there. This folder containing the stories\
+                can be used to load the dataset via `datasets.load_dataset("narrativeqa_manual", data_dir="<path/to/folder>")."""
+
+    def _info(self):
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "kind": datasets.Value("string"),
+                    "story_url": datasets.Value("string"),
+                    "story_file_size": datasets.Value("string"),
+                    "wiki_url": datasets.Value("string"),
+                    "wiki_title": datasets.Value("string"),
+                    "story_word_count": datasets.Value("string"),
+                    "story_start": datasets.Value("string"),
+                    "story_end": datasets.Value("string"),
+                    "story_text": datasets.Value("string"),
+                    "summary": datasets.Value("string"),
+                    "summary_tokenized": datasets.Value("string"),
+                    "qaps": datasets.features.Sequence(
+                        {
+                            "question": datasets.Value("string"),
+                            "answer1": datasets.Value("string"),
+                            "answer2": datasets.Value("string"),
+                            "question_tokenized": datasets.Value("string"),
+                            "answer1_tokenized": datasets.Value("string"),
+                            "answer2_tokenized": datasets.Value("string"),
+                        }
+                    ),
+                }
+            ),
```

Comment:
For consistency I think it could be nice to have the same features here as in narrativeqa, i.e.
```

            features=datasets.Features(
                {
                    "document": {
                        "id": datasets.Value("string"),
                        "kind": datasets.Value("string"),
                        "url": datasets.Value("string"),
                        "file_size": datasets.Value("int32"),
                        "word_count": datasets.Value("int32"),
                        "start": datasets.Value("string"),
                        "end": datasets.Value("string"),
                        "summary": {
                            "text": datasets.Value("string"),
                            "tokens": datasets.features.Sequence(datasets.Value("string")),
                            "url": datasets.Value("string"),
                            "title": datasets.Value("string"),
                        },
                        "text": datasets.Value("string"),
                    },
                    "question": {
                        "text": datasets.Value("string"),
                        "tokens": datasets.features.Sequence(datasets.Value("string")),
                    },
                    "answers": [
                        {
                            "text": datasets.Value("string"),
                            "tokens": datasets.features.Sequence(datasets.Value("string")),
                        }
                    ],
                }
            ),
```
What do you think ?

Reply:
Ok I've copied the same template for it now.